pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -77,67 +77,131 @@ func (msg subscribeMsgType2) String() string {
 }
 
 func (s *Stream) handleConnect() {
-	subProductsMap := make(map[types.Channel][]string)
+	// channel2LocalSymbolsMap is a map from channel to local symbols
+	channel2LocalSymbolsMap := make(map[types.Channel][]string)
 
 	// user data strea, subscribe to user channel for the user order/trade updates
```

Comment:
Please also help fix the typo in the comment: strea"m".

Reply:
Opps, thanks.

Reply:
Done.